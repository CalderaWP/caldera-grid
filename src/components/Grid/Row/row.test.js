import React from 'react';
import renderer from 'react-test-renderer';
import {Row} from "./index";

describe( 'Rows work with columns',  () => {
    it('Works with one column', () => {
        const component = renderer.create(
            <Row
                columns={
                    {
                        'c1': {
                            content: 'HiRoy',
                            uuid: 'c1'
                        }

                    }
                }
                onUpdate={() =>{}}

            />
        );
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

    it('Works with two columns', () => {
        const component = renderer.create(
            <Row
                columns={
                    {
                        'c1': {
                            content: 'HiRoy',
                            uuid: 'c1'
                        },
                        'c2': {
                            content: 'HiMike',
                            uuid: 'c2'
                        }
                    }
                }
                onUpdate={() =>{}}

            />

        );
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });



})
