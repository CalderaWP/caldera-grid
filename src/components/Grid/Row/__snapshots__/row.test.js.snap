// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rows work with columns Works with one column 1`] = `
<div>
  <div />
</div>
`;

exports[`Rows work with columns Works with two columns 1`] = `
<div>
  <div />
  <div />
</div>
`;
